### ISSUE TITLE
Model download links are dead

### ISSUE THREAD

**Issue by elfin888** [geekyutao/Inpaint-Anything#168]
Hi! Thank you so much for creating this fantastic project!! It look absolutely amazing.

I tried to download the pre-trained models but all the Yandex links are dead. Do you happen to have the files hosted anywhere else?

All of these links are dead:
All the data (models, test images, etc.) https://disk.yandex.ru/d/AmdeG-bIjmvSug
Test images from the paper https://disk.yandex.ru/d/xKQJZeVRk5vLlQ
The pre-trained models https://disk.yandex.ru/d/EgqaSnLohjuzAg
The models for perceptual loss https://disk.yandex.ru/d/ncVmQlmT_kTemQ

The best model (Places2, Places Challenge):
curl -L $(yadisk-direct https://disk.yandex.ru/d/ouP6l8VJ0HpMZg) -o big-lama.zip
unzip big-lama.zip

All models (Places & CelebA-HQ):
curl -L $(yadisk-direct https://disk.yandex.ru/d/EgqaSnLohjuzAg) -o lama-models.zip
unzip lama-models.zip

Prepare images and masks
Download test images:
curl -L $(yadisk-direct https://disk.yandex.ru/d/xKQJZeVRk5vLlQ) -o LaMa_test_images.zip
unzip LaMa_test_images.zip

Thanks!

**Comment by sergiyclas** [comment]
Do you find a place where yu can download it? Because I need big-lama.pth, but I can't download it

**Comment by MarkChenYutian** [comment]
Hi @sergiyclas @elfin888 , I found the original download link for the Lama here https://github.com/advimman/lama

Specifically,

```
curl -LJO https://huggingface.co/smartywu/big-lama/resolve/main/big-lama.zip
unzip big-lama.zip
```
